### ISSUE TITLE
Unable to install due to ruamel.yaml error

### ISSUE THREAD

**Issue by brownjohnf** [calebmadrigal/trackerjacker#14]
I've tried to install on both Alpine and Debian, both x86_64, and on both I get the following error:

```
Collecting trackerjacker
  Using cached https://files.pythonhosted.org/packages/d2/81/4dca960aeb78e070c1639ccbacfaa17182762f5c808c2eee0f1906c5fff4/trackerjacker-1.8.7.tar.gz
Collecting scapy==2.4.0 (from trackerjacker)
  Using cached https://files.pythonhosted.org/packages/68/01/b9943984447e7ea6f8948e90c1729b78161c2bb3eef908430638ec3f7296/scapy-2.4.0.tar.gz
Collecting pyaml>=17.12.1 (from trackerjacker)
  Using cached https://files.pythonhosted.org/packages/17/c1/5892f756109e54ed53c753129b0da4acf6b6add8dff5a85b18667553b16d/pyaml-17.12.1-py2.py3-none-any.whl
Collecting ruamel.yaml==0.15.0 (from trackerjacker)
  Using cached https://files.pythonhosted.org/packages/10/aa/f31c8a858e23fe5b14c59e6a03ae80b4cb76353c184df48a582a55eb2674/ruamel.yaml-0.15.0.tar.gz
    Complete output from command python setup.py egg_info:
    Warning: 'keywords' should be a list, got type 'NoneType'
    sys.argv ['-c', 'egg_info', '--egg-base', 'pip-egg-info']
    test compiling test_ruamel_yaml
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-vfflz0kq/ruamel.yaml/setup.py", line 858, in <module>
        main()
      File "/tmp/pip-install-vfflz0kq/ruamel.yaml/setup.py", line 847, in main
        setup(**kw)
      File "/usr/local/lib/python3.7/site-packages/setuptools/__init__.py", line 140, in setup
        return distutils.core.setup(**attrs)
      File "/usr/local/lib/python3.7/distutils/core.py", line 108, in setup
        _setup_distribution = dist = klass(attrs)
      File "/usr/local/lib/python3.7/site-packages/setuptools/dist.py", line 370, in __init__
        k: v for k, v in attrs.items()
      File "/usr/local/lib/python3.7/distutils/dist.py", line 267, in __init__
        getattr(self.metadata, "set_" + key)(val)
      File "/usr/local/lib/python3.7/distutils/dist.py", line 1203, in set_keywords
        self.keywords = _ensure_list(value, 'keywords')
      File "/usr/local/lib/python3.7/distutils/dist.py", line 40, in _ensure_list
        value = list(value)
    TypeError: 'NoneType' object is not iterable
```

I've tried installing libyaml, etc., but it appears to have no effect. This is with an up-to-date pip3.

**Comment by TheT3ngu** [comment]
I get a very similiar error when I run `sudo pip3 install trackerjacker`:
```
Collecting trackerjacker
  Using cached https://files.pythonhosted.org/packages/d2/81/4dca960aeb78e070c1639ccbacfaa17182762f5c808c2eee0f1906c5fff4/trackerjacker-1.8.7.tar.gz
Collecting scapy==2.4.0 (from trackerjacker)
  Using cached https://files.pythonhosted.org/packages/68/01/b9943984447e7ea6f8948e90c1729b78161c2bb3eef908430638ec3f7296/scapy-2.4.0.tar.gz
Collecting pyaml>=17.12.1 (from trackerjacker)
  Using cached https://files.pythonhosted.org/packages/17/c1/5892f756109e54ed53c753129b0da4acf6b6add8dff5a85b18667553b16d/pyaml-17.12.1-py2.py3-none-any.whl
Collecting ruamel.yaml==0.15.0 (from trackerjacker)
  Using cached https://files.pythonhosted.org/packages/10/aa/f31c8a858e23fe5b14c59e6a03ae80b4cb76353c184df48a582a55eb2674/ruamel.yaml-0.15.0.tar.gz
    Complete output from command python setup.py egg_info:
    /tmp/tmp_ruamel_u3yv84mj/test_ruamel_yaml.c: In Funktion »main«:
    /tmp/tmp_ruamel_u3yv84mj/test_ruamel_yaml.c:6:8: Warnung: »parser« wird in dieser Funktion uninitialisiert verwendet [-Wuninitialized]
     parser = parser;  /* prevent warning */
     ~~~~~~~^~~~~~~~
    Warning: 'keywords' should be a list, got type 'NoneType'
    sys.argv ['-c', 'egg_info', '--egg-base', 'pip-egg-info']
    test compiling test_ruamel_yaml
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-ujuemk4h/ruamel.yaml/setup.py", line 858, in <module>
        main()
      File "/tmp/pip-install-ujuemk4h/ruamel.yaml/setup.py", line 847, in main
        setup(**kw)
      File "/usr/lib/python3.7/site-packages/setuptools/__init__.py", line 131, in setup
        return distutils.core.setup(**attrs)
      File "/usr/lib/python3.7/distutils/core.py", line 108, in setup
        _setup_distribution = dist = klass(attrs)
      File "/usr/lib/python3.7/site-packages/setuptools/dist.py", line 370, in __init__
        k: v for k, v in attrs.items()
      File "/usr/lib/python3.7/distutils/dist.py", line 267, in __init__
        getattr(self.metadata, "set_" + key)(val)
      File "/usr/lib/python3.7/distutils/dist.py", line 1203, in set_keywords
        self.keywords = _ensure_list(value, 'keywords')
      File "/usr/lib/python3.7/distutils/dist.py", line 40, in _ensure_list
        value = list(value)
    TypeError: 'NoneType' object is not iterable
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-install-ujuemk4h/ruamel.yaml/
```
I am on Arch.

**Comment by brownjohnf** [comment]
Interesting. I guess I should have clarified that I'm also on Arch, but running the install in a Docker container (tried Alpine and Debian). The container user is root, but it wasn't privileged. I figured I'd get it installed and then run it attached to the host network.

**Comment by TheT3ngu** [comment]
@deltaxflux Thanks, worked fine for me!

**Comment by calebmadrigal** [comment]
Fixed in version 1.8.8.

**Comment by tucked** [comment]
@calebmadrigal What was the fix?

**Comment by tucked** [comment]
Fix is #7

**Comment by tucked** [comment]
Also related: https://bitbucket.org/ruamel/yaml/issues/230/ruamelyaml-015-fails-install-on-python37
